tests/authorizations/test_emergency_contact_authorization.py

```diff
@@ -0,0 +1,43 @@
+from conftest import is_valid
+import pytest
+
+
+@pytest.mark.usefixtures("client_class", "empty_test_db")
+class TestEmergenyContactsAuthorizations:
+    def setup(self):
+        self.endpoint = "/api/emergencycontacts"
+
+    def test_emergency_contacts_POST(self, auth_headers):
+        endpoint = "/api/emergencycontacts"
```

Comment:
I think here, if you want to use `endpoint` instead of `self.endpoint`, then that variable should be defined as 
```python
endpoint = self.endpoint
```
Otherwise there doesn't seem to be any reason to have the `setup()` in there.

Same thing on line 32.